src/reference/modules/installation/pages/_partials/debian_repo.adoc

```diff
@@ -15,14 +15,22 @@ echo "deb http://repository.rudder.io/apt/7.0/ $(lsb_release -cs) main" > /etc/a
 [NOTE]
 ====
 
-If you have an active subscription, use the following to get access to long term support (you need to replace
-the user name and the password by your Rudder account):
+If you have an active subscription, use the following to get access to long term support.
+You need to replace the user name (LOGIN) and the password (PASSWORD) by your Rudder account:
 
 [source, Bash]
 ----
 
 echo "deb https://LOGIN:PASSWORD@download.rudder.io/apt/7.0/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
 
+# for recent debian (>=10) and ubuntu (>=20)
+echo "machine download.rudder.io login LOGIN password PASSWORD" > /etc/apt/auth.conf.d/rudder.conf
+chmod 640 /etc/apt/auth.conf.d/rudder.conf
+
+# on old debian (<10) and ubuntu (<20) use this instead
+#echo "machine download.rudder.io login LOGIN password PASSWORD" >> /etc/apt/auth.conf
```

Comment:
```suggestion
echo "machine download.rudder.io login LOGIN password PASSWORD" >> /etc/apt/auth.conf
```

Reply:
this would make a simple copy paste create 2 authentication files instead of one

Reply:
good ?

Reply:
yes, but we should also comment the second chmod then